server/controllers/finance/vouchers.js

```diff
@@ -193,12 +208,36 @@ function report(req, res, next) {
 }
 
 /**
+* GET /vouchers/reports
+*
+* @method report
+*/
+function report(req, res, next) {
+  'use strict';
+
+  let reportUrl = './server/controllers/finance/reports/voucher.report.handlebars';
```

Comment:
Since these URLs never change, could they be `const`? Can they be outside the function?  That way, the V8 garbage collector doesn't have to set and unset variables every HTTP request that calls this function.